### DISCUSSION TITLE
Customize Form Component Values

### DISCUSSION THREAD

**Discussion by agoe** [refinedev/refine#1568] [General]
In  
https://github.com/pankod/refine/tree/master/examples/customPages/src/pages/post-review
const handleUpdate = (item: IPost, status: string) => {
        mutate({ resource: "posts", id: item.id, values: { ...item, status } });
    };
handleUpdate can easily be adapted .

In https://github.com/pankod/refine/blob/master/examples/tutorial/antd/src/pages/posts/edit.tsx
it is not so obvious.

I added a userId column/attribute in  the posts Interface and filling that with the current userId of the loggedIn user
is a bit of a struggle for me as I do not know how to access the update mutation.

Thanks fore some insights.

Ps.
Adapting the simple rest adapter to spring data  is  up to now quite easy.

I did da similar thing with react-admin and  SqlAlchemy

**Comment by omeraplak** [comment]
Hey @agoe ,
Thank you for contacting us!  
As in our multi-tenancy example, you can use the `onFinish` property of the `<Form>` component,
https://github.com/pankod/refine/blob/master/examples/multi-tenancy/strapi/src/components/product/create.tsx#L60
